### PR TITLE
Bundle all the modules inside the app

### LINKED ISSUE #406: Let all the modules to bundle on the app
Currently, we use a pre-bundled set of modules to load core modules like React, React DOM etc.
With this (specially with built in CDN support) we thought to increase the page load speed.

But it seems like if cause few issues:

* Some modules shared between the pre-packages bundle and the app's bundle (specially, babel and core-js related modules)
* If the user use React like frameworks, this has no use.
* Eventually, everyone will starting to use [`babel-preset-env`](https://github.com/babel/babel-preset-env). So, we never will be able to use the pre-packaged bundle.

So, we are getting rid of that and going back to the normal way how we bundle.

### PR DESCRIPTION
**Pull request by arunoda**
Fixes #406

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,7 +63,6 @@
     "react": "15.4.1",
     "react-dom": "15.4.1",
     "react-hot-loader": "3.0.0-beta.6",
-    "read-pkg-up": "2.0.0",
```
Comment (nkzawa) [inline package.json]: I think we can remove `babel-plugin-transform-remove-strict-mode` too

Comment (arunoda) [inline package.json]: Yep.

Comment (arunoda) [inline package.json]: Done.

**Comment by nkzawa** [comment]
👍 👍 👍 awesome

**Comment by lukeed** [comment]
Thanks for adopting this @arunoda 🙌

**Pull request merged**